legacy/routes/multiapi.js

```diff
@@ -0,0 +1,72 @@
+var express = require('express');
+var router = express.Router();
+var utils = require('../util/utils');
+
+var multiapi = function(optionalCoverage) {
+    optionalCoverage['putTestOneApiVersionOne'] = 0;
```

Comment:
Please prefix them all with `MultiApi`, so it's easier for people that tried to implement it to skip them all